getting-started/troubleshooting/error-handling.rst

```diff
@@ -228,18 +227,49 @@ a mistake was made entering the permalink:
    Warning that permalink was incorrect, allows user to retry upload DIP
 
 
-.. _common-errors:
+.. _failure-errors:
 
-Other common error behaviours
+Errors that fail transfers
 -----------------------------
 
-#. Verify metadata directory checksums: if the checksums in the metadata
-   directory cannot be verified (i.e. if a file is missing or corrupted) the
-   microservice will fail and the transfer will be moved in the failed
+There are a number of microservice errors that will halt the workflow and cause
+Archivematica to report a 'Failed transfer'. It is important to note that not
+all microservice errors will lead to a failed transfer. Below is a list of
+common errors that will result in the 'Failed transfer' microservice to run and
+the transfer to be moved to the failed directory.
+
+#. Approve Transfer (zipped and unzipped bags): When running a zipped or
+   unzipped bag transfer, the first microservice is 'Approve transfer.' If the
+   'verify bag, and restructure for complaince' job fails within this
+   microservice, the transfer will fail and be moved to the failed directory.
+   This will happen when the bag does not conform to the `BagIt`_ specification
+   or when one of the components is incorrect.
+
+#. Verify transfer compliance: if a job fails within this microservice, the
+   transfer will also fail and will be moved to the fail directory.
+
+#. Verify transfer checksums: if the checksums in the metadata
+   directory cannot be verified (i.e. if a file is missing or corrupted) this
+   microservice will fail and the transfer will be moved to the failed
    directory.
 
-#. Scan for viruses: if a virus is found the microservice will fail and the
-   transfer will be moved in the failed directory.
+#. Scan for viruses: if a virus is found in the transfer then the microservice
+   will fail and the transfer will be moved to the failed directory.
+
+#. Generate METS.xml document: if this microservice fails to generate a portion
+   of the METS file when run in either the Transfer Tab or the Ingest Tab, it
+   will fail the transfer or SIP. Either the transfer or SIP will be moved to
+   the failed directory. The error logs will give details about the issue so
+   you can investigate further.
+
+
+.. _common-errors:
+
+Other common error behaviours
+-----------------------------
+
+Below is a list of common errors that, like normalization, will produce an
```

Comment:
```suggestion
Below is a list of common errors that will produce an
```
It could be confusing to refer to normalization here, if someone hasn't read the normalization section above.

Reply:
good point!